test/cql-pytest/conftest.py

```diff
@@ -262,3 +262,8 @@ def has_tablets(cql):
 def skip_without_tablets(scylla_only, has_tablets):
     if not has_tablets:
         pytest.skip("Test needs tablets experimental feature on")
+
+@pytest.fixture(scope="function")
+def xfail_tablets(request, has_tablets):
```

Comment:
The xfail_tablets fixture which you're re-adding here previously existed, but was **deliberately** deleted in commit 7888b23e9e71b42adba1da02b3be795ea73bf834 by @denesb. 

The problem with xfail_tablets was that it loses test converage: This test is run only with tablets (the default), and then skipped - so we lose the ability to catch regression. @avikivity didn't like this. Botond's alternative solution was (e.g., see 3527d0aaedc8bc67b195e230741de6167942f981) to use "parameterize" to run this test twice with and without tablets, and mark the tablets run as xfail and the vnode run without xfail.